sources/packages/web/src/components/common/Reports.vue

```diff
@@ -69,7 +87,10 @@ export default defineComponent({
         loading.value = false;
       }
     };
-    return { exportReport, formLoaded, submitForm, Role, loading };
+    const loadInstitutionReportDetails = async (form: FormIOForm) => {
```

Comment:
Minor, not a blocker, the creation of an additional method just to invoke a one-code method that is called once does not seem to add too much benefit or improve the code readability, IMO.